harness/determined/experimental/_native.py

```diff
@@ -71,3 +72,29 @@ def test_one_batch(
         logging.info(
             "Note: to submit an experiment to the cluster, change local parameter to False"
         )
+
+
+def test_one_batch_pytorch(trial_class: Any, config: Dict) -> Any:
+    try:
+        from determined import pytorch
+    except ImportError:
+        logging.error("unable to import determined.pytorch module")
+        return
```

Comment:
logging an error and exiting successfully is not `--test --local` mode should do